routes/s3upload.js

```diff
@@ -0,0 +1,41 @@
+const s3UploadRouter = require('express-promise-router')();
+const aws = require('aws-sdk');
+const crypto = require('crypto');
+
+// initialize a S3 instance
+const bucketName = process.env.S3_BUCKET_NAME;
+const accessKeyId = process.env.AWS_ACESS_KEY_ID;
+const secretAccessKey = process.env.AWS_SECRET_ACCESS_KEY;
+const region = process.env.AWS_REGION;
```

Comment:
If these are only used once, you can just directly put the env var where it's needed. No need to create additional variables for them.